src/main/java/org/ow2/proactive/catalogclient/util/ResponseUtils.java

```diff
@@ -40,4 +44,8 @@ public static void checkResponse(Response response) {
             throw new FailedRequestException(response.getStatusCode() + " " + response.getStatusLine());
         }
     }
+
+    private static void sss() throws FileNotFoundException {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "sss" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUnusedPrivateMethod)